### PR TITLE
Fixes #30378 - added uploader.sh typealias

### PR DESCRIPTION
**Pull request by lzap**
Please follow the README to test this (the `make` command).

### CODE REVIEW

File: foreman.fc [hunk 1]
```diff
@@ -69,3 +69,6 @@
 
 /usr/share/gems/gems/foreman-tasks-.*/bin/foreman-tasks -- gen_context(system_u:object_r:foreman_tasks_exec_t,s0)
 /opt/theforeman/tfm/root/usr/share/gems/gems/foreman-tasks-.*/bin/foreman-tasks -- gen_context(system_u:object_r:foreman_tasks_exec_t,s0)
+
+# Red Hat Cloud plugin
+/var/lib/foreman/red_hat_inventory/uploads/uploader.sh -- gen_context(system_u:object_r:foreman_uploader_bin_t,s0)
```
Comment (ekohl) [inline foreman.fc:74]: Before we add this, I'd like to investigate alternatives. This morning with @ShimShtein and @adamruzicka we talked about this. Other complications that we may run into is that `/var/lib` might be mounted `noexec` in which case it'd also fail.

Comment (adamruzicka) [reply foreman.fc:74]: What would be the best location for it? It needs to be a place where it makes sense (something like `/usr/libexec`) but at the same time is writable by the rails process.

Comment (ekohl) [reply foreman.fc:74]: My first question is why it needs to be dynamically generated.

**Comment by ekohl** [comment]
The RH Cloud Plugin depends on Katello. Should we keep it in the katello-selinux policy?

File: foreman.fc (hunk 1 above)
Comment (ShimShtein) [reply foreman.fc:74]: @ekohl, it's for disconnected systems. The idea is to generate a script that can be downloaded alongside with the payload and executed with minimal user interaction. For this reason I "hard code" some variables into the script, so the user will not need to input them.

Comment (lzap) [reply foreman.fc:74]: I had the very same question, you can still parametrize a short shell script as needed.

I agree that proper solution is to make this a generic script in `/usr/lib/libexec`.

**Comment by lzap** [comment]
> The RH Cloud Plugin depends on Katello. Should we keep it in the katello-selinux policy?

I can move that, had no idea.

**Comment by lzap** [comment]
I moved this to katello-selinux: https://github.com/Katello/katello-selinux/pull/27

**Pull request closed without merging**